src/inventory-count/application/use-cases/update-inventory-count/update-inventory-count.use-case.ts

```diff
@@ -0,0 +1,55 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { UseCase } from 'src/core/domain/use-case.interface';
+import { InventoryCountRepositoryPort } from 'src/inventory-count/domain/inventory-count-repository.port';
+import { InventoryCountDto as InventoryCountUi } from 'src/inventory-count/presentation/dto/inventory-count.dto';
+import { InventoryCountMapper } from '../../mappers/inventory-count.mapper';
+import { InventoryCountNotFoundError } from 'src/inventory-count/errors/inventory-count-not-found.error';
+import { InventoryCountItemService } from '../../services/inventory-count-item.service';
+import { InventoryCountItemType } from 'src/inventory-count/domain/inventory-count.entity';
+
+interface Props {
+  inventoryCountTemplateId: string;
+  staffName: string;
+  inventoryCountId: string;
+  inventoryCountItems: {
```

Comment:
we should not allow updating template and items. items are created automatically